plugins/modules/server.py

```diff
@@ -451,6 +525,28 @@ def _update_server(self, server_info):
             if desired_server_group_ids != current_server_group_ids:
                 self._module.warn("Server groups can not be mutated, server needs redeployment to change groups.")
 
+        # Remove interface properties that were not filled out by the user
+        self.normalize_interfaces_param()
+
+        # Compare the interfaces as specified by the user, with the interfaces
+        # as received by the API. The structures are somewhat different, so
+        # they need to be evaluated in detail
+        wanted = self._module.params.get('interfaces')
+        actual = server_info.get('interfaces')
+
+        try:
+            update_interfaces = self._module.params.get('force') \
```

Comment:
Why do you unconditionally update the interfaces if `force` is `True`? The documentation says this parameter is to allow stopping running servers to update them if this is necessary (eg. flavor change).

If force updating the interfaces is under some conditions necessary, I would either add a separate parameter for this or at least update the documentation.

Reply:
This can be removed, I wasn't aware of the specific semantics.